app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+  def current_user
+    session[:customer_id].present? ? Customer.find_by(id: session[:customer_id]) : nil
+  end
```

Comment:
You can declare the method defined in application controller as helper method and use that.